### PR TITLE
Remove unneeded files - (this is the former Vuetify PR)

### PR DESCRIPTION
**Pull request by dominikbruechner**
# Short Description

This is the former Vuetify proof of concept PR. 

tbd:

- [ ] Update Vuepress documentation

## Checklist before merging

- [ ] QA: In addition to review, the code has been manually tested (if manual testing is possible)
- [ ] PO: Any deviation from requirements was agreed with Product-Owner / ticket author / support-team

> Notice: Please keep this Pull-Request as a Draft (or add WIP label), until it is ready to be reviewed

### CODE REVIEW

File: src/serverMiddleware/routes.js [hunk 1]
```diff
@@ -33,6 +33,8 @@ const routes = [
 	`^/insights`,
 	// `^/news/${mongoId}/edit`,
 	`^/news/new`,
+	"^/news/",
+	"^/news",
```
Comment (dominikbruechner) [inline src/serverMiddleware/routes.js:37]: Needs to be deactivated before it goes in develop

**Comment by dominikbruechner** [comment]
> ❤️ I have deployed this pull-request for you:
> 
> **Docs:** http://docs.1760.nuxt.schul-cloud.surge.sh
> **Storybook:** http://stories.1760.nuxt.schul-cloud.surge.sh

First Story in Storybook is Vuetify

File: src/pages/news/index.vue [hunk 2]
```diff
@@ -1,33 +1,59 @@
...
-				label: $t('pages.news.new.title'),
-			}"
-		/>
-		<section v-if="news && news.length > 0" class="section">
-			<news-card
+	<v-container>
+		<v-row>
+			<v-col>
+				<h1 class="text-h1">{{ $t("pages.news.title") }}</h1>
+				<p>Lorem Ipsum, normaler Text.</p>
```
[the remark this answers is not in the document]
Comment (dominikbruechner) [reply src/pages/news/index.vue:6]: this is just a demo page for Vuetify, and should probably be removed once the PR is merged

File: src/components/base/BaseTextarea.vue [hunk 3]
```diff
@@ -145,7 +145,7 @@ export default {
 	}
 }
 
-/deep/ .top .input-line {
+::v-deep .top .input-line {
```
[the remark this answers is not in the document]
Comment (dominikbruechner) [reply src/components/base/BaseTextarea.vue:148]: /deep/ is apparently deprecated, and for SASS the recommendation is to use ::v-deep in Vue 2 https://stackoverflow.com/questions/48032006/how-do-i-use-deep-or-or-v-deep-in-vue-js

without SASS we would use >>>

**Comment by dominikbruechner** [comment]
@schul-cloud-bot update screenshots

**Comment by dominikbruechner** [comment]
Regarding the [FIX] removed v-app wrapper from storybook commit:
This removes the v-app wrapper from ALL stories in storybook again. 
Why? It had some side effects on existing components, and is only needed for vuetify stories. 
now screenshot tests are green again

**Comment by dominikbruechner** [comment]
@schul-cloud-bot update screenshots

**Comment by dominikbruechner** [comment]
@schul-cloud-bot update screenshots

**Comment by dominikbruechner** [comment]
@schul-cloud-bot update screenshots

**Comment by dominikbruechner** [comment]
@schul-cloud-bot update screenshots

File: src/themes/base/styles/utility.scss [hunk 4]
```diff
@@ -0,0 +1,59 @@
+@import "./utility/breakpoints";
```
Comment (CeEv) [inline src/themes/base/styles/utility.scss:1]: index file for utility

File: src/layouts/defaultVuetify.vue [hunk 5]
```diff
@@ -0,0 +1,28 @@
+<template>
```
Comment (CeEv) [inline src/layouts/defaultVuetify.vue:1]: This is for what, because the name defaultVuetify.vue is confusing?

**Review by CeEv — APPROVED** [review]

File: src/layouts/defaultVuetify.vue (hunk 5 above)
Comment (dominikbruechner) [reply src/layouts/defaultVuetify.vue:1]: This puts the vuetify application wrapper to the default layout. 

We can think about alternative names

**Review by pkleybolte — DISMISSED** [review]

**Review by pkleybolte — APPROVED** [review]

**Review by odalys-hpi — APPROVED** [review]

**Pull request closed without merging**